### DISCUSSION TITLE
Is there a way to keep the zoom view the same until changed?

### DISCUSSION THREAD

**Discussion by tack32** [smplayer-dev/smplayer#324] [Q&A]
I wanted to crop some sports videos as I watched them. I heard that couldn't be done. I thought I'd found a work around by using zoom. I zoom it in and then use the alt-< or ^ keys etc to center the picture. However, every time I jump forward or backward the picture moves back. Is there a way to fix it in that same zoom position? Thanks.

**Comment by smplayer-dev** [comment]
I'm afraid I can't reproduce this problem.
